src/screens/NewAnalytics/NewAnalyticsUtils.res

```diff
@@ -34,3 +34,34 @@ let fillMissingDataPoints = (
 
   dataPoints->Dict.valuesToArray
 }
+
+open NewAnalyticsTypes
+
+let requestBody = (
+  ~dimensions as _: array<dimension>,
+  ~startTime: string,
+  ~endTime: string,
+  ~metrics: array<metrics>,
+  ~groupByNames: option<array<string>>=None,
+  ~filters as _: option<array<dimension>>=[]->Some,
+  ~customFilter as _: option<dimension>=None,
+  ~excludeFilterValue as _: option<array<status>>=None,
+  ~applyFilterFor as _: option<array<status>>=None,
+  ~delta: option<bool>=None,
+  ~granularity: option<string>=None,
+) => {
+  let metrics = metrics->Array.map(v => (v: metrics :> string))
+  let filter = Dict.make()->JSON.Encode.object->Some
+
+  [
+    AnalyticsUtils.getFilterRequestBody(
```

Comment:
move this function inside NewAnalyticsUtils

Reply:
its there in NewAnalyticsUtils utself